### ISSUE TITLE
ReadTheDocs thinks 3.15.0 is still the latest version

### ISSUE THREAD

**Issue by adamnovak** [DataBiosphere/toil#2815]
If you go to https://toil.readthedocs.io/en/3.14.0/developingWorkflows/toilAPI.html, ReadTheDocs helpfully tries to send you to the latest version instead. But it sends you to 3.15.0, when really it should be... releases-3.19.0. Or maybe the (missing) releases-3.20.0.

┆Issue is synchronized with this [Jira Task](https://ucsc-cgl.atlassian.net/browse/TOIL-433)
┆Issue Number: TOIL-433

**Comment by DailyDreaming** [comment]
This should probably be a larger ticket for rebuilding and reorganizing any missing release docs.  I noticed that 3.20.0 doesn't exist either.  Release branches weren't consistently named for a while, so it might make sense to go in and make sure that they are.

**Comment by adamnovak** [comment]
I think since we (re)started adding "releases-" to the tag name, ReadTheDocs has been unable to parse our version tags. It hasn't been activating new versions itself, which it should be able to do. I will see if I can explain to it how to do that.

It also hasn't been building master on every commit, though. Maybe we're missing a hook?

**Comment by adamnovak** [comment]
OK, I've gone and added an automation rule to activate all the "releases" tags as versions automatically, and a web hook on the Toil repo to keep the master build up to date with the actual master. Whenever it builds master it detects all the other new tags and builds them, so I think this should keep things in sync.

I also deactivated/hid some of the old versions' docs that didn't appear to actually exist. If you clicked on them in the version picker you would just get RtD's "maze found" page.

I think this has been fixed. RtD still sorts the plain-number tags first in the version picker, but there's not much we can do about that unless we deactivate the docs pages for all of them.
